common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/ImageSource.scala

```diff
@@ -1,11 +1,20 @@
 package uk.ac.wellcome.models.work.internal
 
-sealed trait ImageSource[Id <: WithSourceIdentifier] {
+sealed trait ImageSource[Id <: WithSourceIdentifier, DataId <: IdState] {
   val id: Id
   val ontologyType: String
 }
 
-case class SourceWork[Id <: WithSourceIdentifier](
+case class SourceWorks[Id <: WithSourceIdentifier, DataId <: IdState](
+  canonicalWork: SourceWork[Id, DataId],
+  redirectedWork: Option[SourceWork[Id, DataId]]
```

Comment:
What if we have multiple redirected works?

Reply:
I think `redirectedWork` here refers to the work that the image was actually from, but is now redirected (eg, the Miro work that was merged with a Sierra work)

Reply:
We had this discussion with @jamieparkinson but in practice, it's always going to be one or 2 works, not more. If that changes we can change the model

Reply:
Ah yes, OK, that makes sense